message_ix/tests/test_soft_constraints.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import pandas as pd
+import pandas.testing as pdt
+
+from message_ix.testing import make_westeros
+
+
+def test_soft_constraint(test_mp):
+    s = make_westeros(test_mp, quiet=True)
```

Comment:
I think it would be easier to assert that the numbers are correct if one would use a simpler test model - one region, one commodity, two technologies with very obvious numbers...

Reply:
The test has been adjusted accordingly.